cfbs/main.py

```diff
@@ -127,7 +127,7 @@ def main() -> int:
         _get_arg_parser().print_help()
         return 0
 
-    config = CFBSConfig.get_instance(args.index, args.non_interactive)
+    _ = CFBSConfig.get_instance(args.index, args.non_interactive)
```

Comment:
We don't even need to assign the instance to a variable, right?

Reply:
It is needed for tests to pass, but I haven't investigated why

Reply:
That's weird, it shouldn't be needed.

Reply:
Here we create the instance with arguments
```python3
_ = CFBSConfig.get_instance(args.index, args.non_interactive)
```

All other places we just get the already created instance *_(as it is a singleton)_* and don't pass any arguments
```python3
config = CFBSConfig.get_instance()
```
Thus we need it 😉

Reply:
So what exactly fails and how? I don't think you are required to assign the returned value to a variable in Python if you don't care about it.

Reply:
I.e. the tests halt because the "--non-interactive" option is not passed to the config.